.github/workflows/wheel.yaml

```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-18.04, windows-latest, macos-latest]
+        os: [ubuntu-18.04, ubuntu-20.04, windows-latest, macos-latest]
```

Comment:
Is it really necessary, or can we build wheel only on one of the two linux ?

Reply:
I don't think both are needed. So I will just replace with `ubuntu-latest`.